docs/Administration.rst

```diff
@@ -2247,6 +2247,33 @@ By default, all existing groups will be displayed on the left panel. The
 right panel shows information of the selected group and of its members
 with the Add Member form.
 
+.. _ExistingGroups:
+
+**Existing Groups**
+
+By default, in eXo Platform there are 4 parent groups: Development, Organzation, Platform and Spaces.
+
+**Development** is a sample group, could be used to organize users.
+
+**Organization** helps you to organize your company: deparments, teams or any way to represent your organization.
+
+.. tip:: Notice that these groups i.e. those under Organization group have no specific effects in the product.
+         They are just made as a placeholder.
+
+**Platform** helps to define roles in the platform, it contains 4 sub groups:
+-  Administration: contains users having administrative roles i.e. who are able to access the Administration menu |imagee354|.
+-  Guests: contains external users to the platform, they have access only to public content of the platform
```

Comment:
If I remember, no user is really assigned this group. It’s a membership automatically assigned to non authenticated users and thus, this technical group is mandatory to set permissions on this visible by non authenticated users only (such as pages and content)